wallet/wallet-rpc-lib/src/rpc/server_impl.rs

```diff
@@ -609,18 +597,14 @@ where
         change_address: Option<RpcAddress<Destination>>,
         options: TransactionOptions,
     ) -> rpc::RpcResult<ComposedTransaction> {
-        let config = ControllerConfig {
-            in_top_x_mb: options.in_top_x_mb(),
-            broadcast_to_mempool: true,
-        };
         rpc::handle_result(
             self.request_send_coins(
                 account_arg.index::<N>()?,
                 address,
                 amount,
                 selected_utxo.into_outpoint(),
                 change_address,
-                config,
+                options.into(),
```

Comment:
There is a bunch of functions in this file - `transaction_from_cold_input`, `decommission_stake_pool_request`, `make_tx_for_sending_tokens_with_intent`, `make_tx_to_send_tokens_from_multisig_address` and `create_htlc_transaction` (if I haven't missed one) that never attempt to broadcast the tx, so passing `TransactionOptions` to them, which now has the `broadcast_to_mempool` field, looks confusing. Perhaps we should have a separate type (e.g. named `TransactionRequestOptions`) without the `broadcast_to_mempool` field?

Reply:
fixed and also changed the rpc interface for those methods